metadata-ws/src/test/resources/features/accession.feature

```diff
@@ -0,0 +1,147 @@
+Feature: accession object
```

Comment:
Could you please normalize the style in the feature files?

* Some lines use "Scenario" and others "Scenario outline" (the former is more common) 
* Insert one blank line before each "given-when-then" for grouping, right now it seems a bit arbitrary
* Insert 2 blank lines before each "Scenario" clause to represent a greater split than in the "given-when-then"

Reply:
- Only `Scenario Outline` can have examples section where we can provide parameters for a repetition(loop)
- One blank like after "given-when-then" : done
- 2 blank lines before each "Scenario" : done, had been missed earlier. Corrected everywhere